src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -296,648 +454,673 @@ def build_process_costs(
 
-            self.field_expenses_cost_eq = Constraint(rule=field_expenses_cost_rule)
+                def project_management_and_construction_cost_rule(self):
+                    return self.project_management_and_construction_costs == (
+                        self.total_BEC
+                        * self.project_management_and_construction_percentage
+                    )
 
-            def project_management_and_construction_cost_rule(self):
-                return self.project_management_and_construction_costs == (
-                    self.total_BEC * self.project_management_and_construction_percentage
+                self.project_management_and_construction_cost_eq = Constraint(
+                    rule=project_management_and_construction_cost_rule
                 )
 
-            self.project_management_and_construction_cost_eq = Constraint(
-                rule=project_management_and_construction_cost_rule
-            )
+                def epcm_cost_rule(self):
+                    return self.epcm_costs == (
+                        self.equipment_installation_costs
+                        + self.field_expenses_costs
+                        + self.project_management_and_construction_costs
+                    )
 
-            def epcm_cost_rule(self):
-                return self.epcm_costs == (
-                    self.equipment_installation_costs
-                    + self.field_expenses_costs
-                    + self.project_management_and_construction_costs
-                )
+                self.epcm_cost_eq = Constraint(rule=epcm_cost_rule)
 
-            self.epcm_cost_eq = Constraint(rule=epcm_cost_rule)
+                # rules for calculating Contingency costs
+                def process_contingency_cost_rule(self):
+                    return self.contingency_costs == (
+                        self.total_BEC * self.process_contingency_percentage
+                    )
 
-            # rules for calculating Contingency costs
-            def process_contingency_cost_rule(self):
-                return self.contingency_costs == (
-                    self.total_BEC * self.process_contingency_percentage
+                self.process_contingency_cost_eq = Constraint(
+                    rule=process_contingency_cost_rule
                 )
 
-            self.process_contingency_cost_eq = Constraint(
-                rule=process_contingency_cost_rule
-            )
+                def contingency_cost_rule(self):
+                    return self.contingency_costs == (self.process_contingency_costs)
 
-            def contingency_cost_rule(self):
-                return self.contingency_costs == (self.process_contingency_costs)
+                self.contingency_cost_eq = Constraint(rule=contingency_cost_rule)
 
-            self.contingency_cost_eq = Constraint(rule=contingency_cost_rule)
+                def total_installation_cost_rule(self):
```

Comment:
A minor suggestion, but the `@m.Constraint` decorator generally results in more compact and readable code.